instagram-app/src/components/PostContainer/PostsPage.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PostContainer from './PostContainer'
+import SearchBar from '../SearchBar/SearchBar'
+
+import dummyData from '../../dummy-data';
+
+
+
+
+class PostsPage extends React.Component {
+  constructor(){
+    super();
+    this.state = {
+      dummy: [],
+      searchPosts: []
+    };
+
+  }
+  componentDidMount(){
+    this.setState({dummy:dummyData});
+    this.setState({searchPosts:dummyData});
+  }
+searchHandler = event =>{
+  let Sdummy = this.state.dummy.filter(sUser =>{
+    if (sUser.username.includes(event.target.value)){
+      return sUser;
+    }
+  });
```

Comment:
Good use of the `.includes` method for search filtering here - if the search input text is blank, it will effectively render all posts (using `.includes("")` on any string will always evaluate to `true`).